.pathplanner/settings.json

```diff
@@ -2,8 +2,12 @@
   "robotWidth": 0.86,
   "robotLength": 0.99,
   "holonomicMode": false,
-  "pathFolders": [],
-  "autoFolders": [],
+  "pathFolders": [
```

Comment:
This is how pathplanner stores folders - it stores them in this settings.json file and as fields in the auto and path files.